boxes.lisp

```diff
@@ -350,16 +350,26 @@ N.B., Successive calls may return the same object."
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;; Row Boxes ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(deftype vertical-alignment ()
+  "The alignments available for ROW-BOX."
+  '(member :top :baseline :middle :bottom))
+
 (defclass row-box (box)
   ((padding :initarg :padding
             :type (integer 0)
             :accessor row-box-padding)
    (contents :initarg :contents
-             :accessor row-box-contents))
+             :accessor row-box-contents)
+   (align :initarg :align
+          :accessor row-box-align
+          :type (verticle-alignment)
```

Comment:
just say `vertical-alignment` not `(vertical-alignment)` (though the parenthesized version _is_ valid)